src/components/MentorsList/MentorsList.js

```diff
@@ -1,43 +1,36 @@
-import React, { Component } from 'react';
+import React, { useState, useEffect } from 'react';
 import classNames from 'classnames';
 import InfiniteScroll from 'react-infinite-scroller';
-
 import Card from '../Card/Card';
 
 import './MentorList.css';
 import { Loader } from '../Loader';
 
 const itemsInPage = 20;
 
-export default class MentorsList extends Component {
-  state = {
-    page: 1,
-    ready: false,
-  };
+const MentorsList = props => {
+  const [page, setPage] = useState(1);
+  const [ready, setReady] = useState(false);
+
+  useEffect(() => {
+    setPage(1);
+    setReady(props.ready);
+  }, [props.mentors, props.ready]);
 
-  loadMore = () => {
-    this.setState({
-      page: this.state.page + 1,
-    });
+  const loadMore = () => {
+    setPage(page + 1);
```

Comment:
Please use the function update, since it depends on the previous state, something like `setPage(prevPage => prevPage + 1)` :)

Reply:
My bad, I didn't get to this 😞